lib/testing/testDataGenMissingFrames.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file
+ * @brief FRB beamformer CPU verification
+ */
+
+#ifndef TEST_DATA_GEN_MISSING_FRAMES_HPP
+#define TEST_DATA_GEN_MISSING_FRAMES_HPP
+
+#include "Stage.hpp"
+#include "buffer.h"
+
+/**
+ * @class testDataGenMissingFrames
+ * @brief CPU verification for FRB beamformer
+ *
+ * This is CPU only pipeline to verify the FRB beamformer.
+ * This CPU pipeline does the equivalent of:
+ *   - hsaBeamformReorder
+ *   - name: hsaBeamformKernel
+ *   - name: hsaBeamformTranspose
+ *   - name: hsaBeamformUpchan
+ *
+ * @author Cherry Ng
+ **/
```

Comment:
The comments here don't match what this stage actually does...  Also I think Richard and/or Davor worked on a similar stage for testing their systems with dropped frames.   Can you check if there is any duplication going on here?    Finally the name of this `stage` also doesn't really match what it's doing - it's dropping frames based on a pattern.

Reply:
I gave Davor this stage and he has modified for his needs. I will remove my version of it and use Davor's instead.